content/about/_index.md

```diff
@@ -18,7 +18,9 @@ During the semester, our primary events include
 
 Near the start of each calendar year, we also run [Hack&amp;Roll](https://hacknroll.nushackers.org/), the largest student-run hackathon in Singapore. Hack&Roll is unique for being one of the few hackathons, if not the only, that do not have a set topic for participants; instead, the aim is to simply encourage people to come and have fun building something.
 
-We also run smaller events for the NUS community, like Project Intern, where students who have interned share with others their experience and the benefits they have gained from interning, in a bid to encourage others to intern. Once in a while we also initiate other events like [OpenHack](https://openhack.nushackers.org). Everything we do is ultimately aimed at spreading the hacker culture.
+We also run smaller events for the NUS community, like Project Mentor, where seniors mentor freshmen to help them navigate their first year of university; and Project Intern, where students share their internship experiences and advice to help (mostly) Year 2 and Year 3 students. 
```

Comment:
Find it slightly odd to call Project Mentor an event since its not a 1 day thing. Maybe rename events -> initiatives

Also I think "initiate other events like OpenHack" doesn't stand well as its own paragraph, might be better to merge or delete.

Reply:
Fixed!